src/counter.rs

```diff
@@ -138,15 +137,14 @@ impl LocalCounter {
         }
     }
 
-    /// `inc_by` increments the given value to the local counter. Error if the value is <
-    /// 0.
+    /// `inc_by` increments the given value to the local counter. Panics if
+    /// the value is < 0.
```

Comment:
Could you document it as a panic section? 
https://doc.rust-lang.org/book/first-edition/documentation.html#special-sections